### ISSUE TITLE
Host for Package Control

### ISSUE THREAD

**Issue by lipkau** [PavelJaros/ABAP-Sublime-Plugin#5]
This Package should be made available for installation using Package Control.
I must be hosted at: https://packagecontrol.io

**Comment by fareez-ahamed** [comment]
Yes please :+1:

**Comment by avmotolin** [comment]
Yes, please host it at Package Control (https://packagecontrol.io).
